src/index.js

```diff
@@ -79,6 +84,7 @@ module.exports = (options) => {
         proxyUrl,
         payload,
         stream,
+        useBrotli,
```

Comment:
Let's add all the defaults here as default argument values instead of keeping them in a separate file so it's immediately visible what the defaults are.